pages/common/git-pull.md

```diff
@@ -0,0 +1,11 @@
+# git pull
+
+> Fetch branch from a remote repo and merge it to current repo
+
+- Download the remote-tracking branch from default remote and merge it
+
+`git pull`
+
+- Download changes and merge into HEAD
```

Comment:
I believe this description could apply just as well to plain `git pull`. Shouldn't it mention the `{{branch}}`?